instrumentation/rack/test/opentelemetry/instrumentation/rack/middlewares/tracer_middleware_test.rb

```diff
@@ -113,6 +113,20 @@
         end
       end
 
+      describe 'when a regexp is passed in' do
+        let(:config) { { untraced_endpoints: %r{\A/ping/?} } }
+
+        it 'traces everything' do
```

Comment:
This seems like an odd test name...

Reply:
I've copy-pasted this from a related spec immediately above.